docs/pages/eas-update/getting-started.mdx

```diff
@@ -47,83 +52,177 @@ To configure your project, run the following commands in the order they are spec
     '# Install the latest `expo-updates` library',
     '$ npx expo install expo-updates',
     '',
+    '# Initialize the app in EAS',
+    '$ eas init',
+    '',
     '# Initialize your project with EAS Update',
     '$ eas update:configure',
     '',
-    '# Set up the configuration file for builds',
-    '$ eas build:configure',
   ]}
-  cmdCopy="expo install expo-updates && eas update:configure && eas build:configure"
+  cmdCopy="npx expo install expo-updates && eas init && eas update:configure"
 />
 
-After running these commands, **eas.json** file will be created in the root directory of your project.
-
-Inside the `preview` and `production` build profiles in **eas.json**, add a `channel` property for each:
-
-```json eas.json
-{
-  "build": {
-    "preview": {
-      "channel": "preview"
-      // ...
-    },
-    "production": {
-      "channel": "production"
-      // ...
+Inside **app.json**, make changes to use a fixed `runtimeVersion` property, and to add a request header that the app will use to specify a channel when downloading updates from EAS.
+
+{/* prettier-ignore */}
+```json app.json
+  "expo": {
+    /* @hide ... your existing configuration */ /* @end */
+    "updates": {
+      /* @hide ... */ /* @end */
+      "requestHeaders": {
+        "expo-channel-name": "main"
+      }
+      /* @hide ... */ /* @end */
     }
+    /* @hide ... */ /* @end */
+    runtimeVersion: '1.0.0',
+    /* @hide ... */ /* @end */
   }
-}
 ```
 
 The `channel` allows you to point updates at builds. For example, if we set up a GitHub Action to publish changes on merge, it will make it so that we can merge code into the "production" Git branch. Then, each commit will trigger a GitHub Action that will publish an update that will be available to builds with the channel "production".
 
-> **Optional**: If your project is a bare React Native project, see [Updating bare app](/bare/updating-your-app) for any additional configuration.
-
 </Step>
 
 <Step label="4">
-## Create a build for the project
+## Build the project
```

Comment:
I think we should keep pointing people towards EAS build. The previous version of this guide didn't require android studio / xcode to be set up to use EAS update and that seems like something we should strive to keep.

Reply:
We did keep it -- that guide is on the next page.

This is to get people to try out updates quickly on their desktop without waiting for EAS build.

Reply:
Per our discussion, EAS build guide will remain the "Getting Started" guide, and this new guide will go after the debugging page.